src/main/java/bio/overture/riff/service/RiffService.java

```diff
@@ -75,7 +75,7 @@ public RiffResponse makeRiff(JWTUser user, ShortenRequest request) {
   }
 
   public boolean deleteRiff(JWTUser user, String id) {
-    val optionalRiff = repository.findById(Long.valueOf(id, 36));
+    val optionalRiff = repository.findByUidAndId(user.getUid(), Long.valueOf(id, 36));
     if (optionalRiff.isPresent()) {
       val riff = optionalRiff.get();
       if (user.getUid().equals(riff.getUid())) {
```

Comment:
If we are retrieving this riff with the uid as part of the query, do we still need this conditional here?